### DISCUSSION TITLE
Tally (desktop app for Linux)

### DISCUSSION THREAD

**Discussion by cassidyjames** [plausible/analytics#4141] [Anything Else]
Hey all! I hope this is an okay place to share this, but for the past few years I've been developing a Linux desktop companion app for Plausible called [Tally](https://flathub.org/apps/com.cassidyjames.plausible)!

I figured if anyone in the community is using both Plausible and Linux, you might find it useful. 😄 I'm also always happy for anyone to test and especially contribute to the app if you're so inclined; it's completely [open source on GitHub](https://github.com/cassidyjames/tally).

![Screenshot of Tally showing a Plausible dashboard](https://raw.githubusercontent.com/cassidyjames/tally/d1723acc47ff79df9764aeacd16eeb6091934cb2/data/screenshots/dashboard-light.png)

I just released a new version with a bit of a refreshed look, and I plan to support translations and external links in the next release.

I'd love to hear your thoughts and any feedback!

**Comment by ruslandoga** [comment]
👋 @cassidyjames 

Maybe it could be added to Plausible docs somewhere around [here?](https://plausible.io/docs/integration-guides#ios-apps-and-widgets-for-quick-overview-of-your-stats) 

Relevant file in the docs repo: https://github.com/plausible/docs/blob/master/docs/integration-guides.md

**Reply by cassidyjames** [reply]
Good call! I've filed a PR: https://github.com/plausible/docs/pull/506
